fedimint-cli/src/ng.rs

```diff
@@ -236,6 +259,10 @@ struct InfoResponse {
     denominations_msat: TieredSummary,
 }
 
+pub fn parse_operation_id(s: &str) -> anyhow::Result<OperationId> {
+    OperationId::from_hex(s).map_err(Into::into)
+}
+
```

Comment:
Made redundant by 65f07cc44b4f150001740e90e99af33bf9b13d0c in #2416. Should I make it a separate PR or can we merge that first and then you don't need this fn anymore?

Reply:
I'll review that PR and let's merge that first

Reply:
Rebased and removed this function